content/docs/rfcs/32/README.md

```diff
@@ -0,0 +1,560 @@
+---
+slug: 32
+title: 32/RLN
+name: Rate Limit Nullifier specification
+status: raw
+editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
+contributors: 
+---
+
+# Abstract
+
+The following document covers the specification of the RLN construct as well as the auxiliary libraries necessary for interaction with it.
```

Comment:
```suggestion
The following specification covers the RLN construct as well as some auxiliary libraries useful for interacting with it.
```

Reply:
> Rate limiting nullified (RLN) is a construct based on zero-knowledge proofs that provides an anonymous rate-limited signaling/messaging framework suitable for decentralized (and centralized) environments. 

I wonder if this doesn't belong in abstract aka "what is this", then motivation section is more "why do we need this"